drivers/postgres_test.go

```diff
@@ -0,0 +1,27 @@
+package drivers
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestPostgres_MapField(t *testing.T) {
+	var scenarios = []struct {
+		input  FieldDescriptor
+		output Field
+	}{
+		{
```

Comment:
Please cover more cases

Reply:
3-4 more should be fine at the moment. :)

Reply:
Added a few :grin: